rest-api/resources/src/main/resources/openapi/deviceAsset/deviceAsset-scopeId-deviceId.yaml

```diff
@@ -68,6 +70,22 @@ paths:
             application/json:
               schema:
                 $ref: './deviceAsset.yaml#/components/schemas/deviceAssetDefinitions'
+              example:
```

Comment:
Please remove this example which overrides the one of `deviceAsset.yaml#/components/schemas/deviceAssetDefinitions`